blocks/block-alignment-toolbar/index.js

```diff
@@ -25,6 +25,10 @@ const BLOCK_ALIGNMENTS_CONTROLS = {
 		icon: 'align-full-width',
 		title: __( 'Full width' ),
 	},
+	fixed: {
```

Comment:
I gave George the go-ahead to merge this without this fixed position toggle (which we just removed). I think it's good to merge this block, and we can add the toggle separately if we desire it.